netlify/constants.ts

```diff
@@ -5,13 +5,20 @@ import {
   chainInfo,
 } from "@govtechsg/tradetrust-utils/constants/supportedChains";
 
-export const ALLOWED_ORIGINS = [
-  "http://127.0.0.1:3000",
-  "http://localhost:3000",
-  "https://creator.tradetrust.io",
-  "https://dev.tradetrust.io",
-  "https://tradetrust.io",
-];
+export const ALLOWED_ORIGINS =
+  process.env.NODE_ENV === "test"
+    ? [
+        "http://127.0.0.1:3000",
+        "http://localhost:3000",
+        "https://creator.tradetrust.io",
+        "https://dev.tradetrust.io",
+        "https://tradetrust.io",
```

Comment:
im keeping these cos of tests https://github.com/TradeTrust/tradetrust-functions/blob/fd391e4be5384fad269a8336021a4ae3a46719e1/tests/integration/headers.test.ts#L25-L60